platform/positionmanager/pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "positionmanager"
 version = "20240903.2"
-description = "Service to handle trading based off of upstream model predictions"
+description = "Add your description here"
```

Comment:
Looks like this incorrectly reverted back.

Reply:
fixed